packages/gnome_keyring.rb

```diff
@@ -0,0 +1,37 @@
+require 'package'
+
+class Gnome_keyring < Package
+  description 'GNOME password and secret manager'
+  homepage 'https://www.gnome.org'
+  version '3.36.0'
+  compatibility 'all'
+  source_url 'https://ftp.gnome.org/pub/GNOME/sources/gnome-keyring/3.36/gnome-keyring-3.36.0.tar.xz'
+  source_sha256 'a264b57a8d1a71fdf0d66e8cd6033d013fb828be279c35766545eb9bb3734f87'
+
+  binary_url ({
+     armv7l: 'https://www.dropbox.com/s/pk5kig2f023dsit/gnome_keyring-3.36.0-chromeos-armv7l.tar.xz',
+  })
+  binary_sha256 ({
+     armv7l: '4ef943dcc583e8c0dfed3607323bf8a0bde34546db63cd48fe219e0aca6e79c0',
+  })
+
+  depends_on 'gcr'
+  depends_on 'libcap'
+  depends_on 'libcap_ng'
+  depends_on 'openpam'
+  depends_on 'dconf'
+  depends_on 'gcr'
+  depends_on 'docbook_xsl'
+  depends_on 'glib'
+  depends_on 'libxslt'
+  depends_on 'openssh'
+
+  def self.build
+      system "xmlcatalog /usr/local/etc/xml/catalog" # Fix dox-xml resovle issues
```

Comment:
`system "xmlcatalog #{CREW_PREFIX}/etc/xml/catalog"`?

Reply:
This doesn't need to actually be there, I just haven't removed it yet